charts/spire-agent/README.md

```diff
@@ -0,0 +1,26 @@
+# spire-agent
```

Comment:
Why are we adding this file?

Reply:
The master level repo has a bug; there are things without documenting; it is for this reason that the ts-docs fails; because he diffs and hopes there are no pending helm-docs;

This pr not only adds sql-exporter , it also heals the entire CI of the repo.
ref: https://github.com/ManagedKube/helm-charts/blob/10475b8dfa4a5834c6b88a0347fb9d7d501fee89/.github/helm-docs.sh#L12

Reply:
ok